examples/custom-datagrid-column/README.md

```diff
@@ -1,5 +1,7 @@
-# Custom datagrid column example
+# Custom datagrid column
```

Comment:
Do we need this? This is not accessible from the docs and is a duplicate of the how-to guide to customize datagrids in the new docs

Reply:
As we discussed, only a selected few will be available in the docs. Can we link the how-to guide to this folder?
I have kept the dog app here only because we have mentioned it in the docs.

Reply:
https://deploy-preview-2082--mui-toolpad-docs.netlify.app/toolpad/how-to-guides/customize-datagrid/ we can add a link to this app from this page. We should then match the example showcased in those docs with this example app.

Reply:
The link to Codesandbox in the README at https://github.com/mui/mui-toolpad/tree/master/examples/custom-datagrid-column is failing for me https://codesandbox.io/p/sandbox/github/mui/mui-toolpad/tree/master/examples/custom-datagrid-column. Does this work for you @prakhargupta1?

Reply:
I've changed the example and it seems to work https://jt4oxx-3000.csb.app/_toolpad/app/pages/uVF80NP

Reply:
This link: https://codesandbox.io/p/sandbox/github/mui/mui-toolpad/tree/master/examples/custom-datagrid-column
from production is not working.